thunder/benchmarks/__init__.py

```diff
@@ -2877,6 +2878,99 @@ def fn(self) -> Callable:
         return model
 
 
+class MoEBenchmark(Benchmark, metaclass=UserFacingBenchmarkMeta):
```

Comment:
Let's rename it to include "DeepSeek" and maybe the name of the project from which the code is adapted.